src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -29,14 +29,42 @@
 import com.puppycrawl.tools.checkstyle.utils.CommonUtil;
 
 /**
- * Checks for fall through in switch statements
- * Finds locations where a case <b>contains</b> Java code -
- * but lacks a break, return, throw or continue statement.
+ * <p>
+ * Checks for fall-through in {@code switch} statements.
+ * Finds locations where a {@code case} <b>contains</b> Java code but lacks a
+ * {@code break},{@code return},{@code throw} or {@code continue} statement.
```

Comment:
The system is removing the space between `,` and `{`.
Can this be fixed?

Reply:
Done. Moved to separate commit due to significant amount of changes
b833ad1fe78985d703b8db48cbcb73351695cc75

Reply:
Thanks.